debian/changelog

```diff
@@ -1,3 +1,9 @@
+unattended-upgrades (1.14+nmu1) unstable; urgency=low
+
+  * Allow package downgrade if Pin-Priority exceeds 1000
+
+ -- Kaupo Arulo <kaupoa@microsoft.com>  Tue, 08 Oct 2019 13:00:22 +0300
+
```

Comment:
Please drop the change to the changelog, also please don't NMU it.

Reply:
Done